wire-ios/Wire-iOS/Sources/UserInterface/E2EIdentity/E2EINotificationActionsHandler.swift

```diff
@@ -19,35 +19,38 @@
 import Foundation
 import WireSyncEngine
 
-public protocol E2eINotificationActions {
+public protocol E2EINotificationActions {
 
     func enrollCertificate()
     func snoozeReminder() async
 
 }
 
-final class E2eINotificationActionsHandler: E2eINotificationActions {
+final class E2EINotificationActionsHandler: E2EINotificationActions {
 
     // MARK: - Properties
 
-    private var enrollCertificateUseCase: EnrollE2eICertificateUseCaseInterface?
+    private var enrollCertificateUseCase: EnrollE2EICertificateUseCaseInterface?
     private var snoozeCertificateEnrollmentUseCase: SnoozeCertificateEnrollmentUseCaseProtocol?
+    private var stopCertificateEnrollmentSnoozerUseCase: StopCertificateEnrollmentSnoozerUseCaseProtocol?
 
     // MARK: - Life cycle
 
-    init(enrollCertificateUseCase: EnrollE2eICertificateUseCaseInterface?,
-         snoozeCertificateEnrollmentUseCase: SnoozeCertificateEnrollmentUseCaseProtocol?) {
+    init(enrollCertificateUseCase: EnrollE2EICertificateUseCaseInterface?,
+         snoozeCertificateEnrollmentUseCase: SnoozeCertificateEnrollmentUseCaseProtocol?,
+         stopCertificateEnrollmentSnoozerUseCase: StopCertificateEnrollmentSnoozerUseCaseProtocol?) {
         self.enrollCertificateUseCase = enrollCertificateUseCase
         self.snoozeCertificateEnrollmentUseCase = snoozeCertificateEnrollmentUseCase
+        self.stopCertificateEnrollmentSnoozerUseCase = stopCertificateEnrollmentSnoozerUseCase
     }
 
     public func enrollCertificate() {
         // TODO: [WPB-5496] enroll certificate
-        snoozeCertificateEnrollmentUseCase?.stop()
+        stopCertificateEnrollmentSnoozerUseCase?.invoke()
     }
 
     public func snoozeReminder() async {
-        await snoozeCertificateEnrollmentUseCase?.start()
+        await snoozeCertificateEnrollmentUseCase?.invoke()
```

Comment:
something to think about (non-blocking): does it make sense to have the actionsHandler without the useCases?

 I guess it's beceause useCases are optional from userSession... but do we at least want to know that the function fails in case the useCases are missing.

Reply:
I added logs